app/src/main/java/com/keylesspalace/tusky/components/account/AccountViewModel.kt

```diff
@@ -76,7 +70,7 @@ class AccountViewModel @Inject constructor(
 
     private fun obtainAccount(reload: Boolean = false) {
         if (_accountData.value == null || reload) {
-            isDataLoading = true
+            _refreshState.value = RefreshState.REFRESHING
```

Comment:
```suggestion
            if (reload) {
                _refreshState.value = RefreshState.REFRESHING
            }
```

Reply:
I didn't notice here is the key point, can we revert 3d283bd5b97ff9a06ed6b5fa189de6e2a1662445 and use your logic here, it's a simple state.

```kt
  if (reload) {
      _refreshState.value = true
  }
```

Reply:
6e48b6ad793ab8efd74a684d47403c6645139484